addons/runtime_debug_tools/scripts/debug_runtime_interaction_2d.gd

```diff
@@ -66,6 +83,10 @@ func _ready():
     _ignore_nodes(get_parent())
     
 func _process(_delta: float) -> void:
+    if _is_game_process_in_focus and _set_initial_mouse_mode:
```

Comment:
I think you can remove the manual focus checking you're doing here, and in `_notification`, and replace it with:

`get_window().has_focus()`